s2fft/transforms/spherical.py

```diff
@@ -582,6 +588,8 @@ def forward_jax(
         L_lower (int, optional): Harmonic lower-bound. Transform will only be computed
             for :math:`\texttt{L_lower} \leq \ell < \texttt{L}`. Defaults to 0.
 
+        use_cuda (bool, optional): Whether to use the CUDA backend. Defaults to False.
+
```

Comment:
I think it would be worth expanding on what this flags does a bit more in this docstring and/or using a more informative name. My understanding is this only has an effect when `sampling == "healpix"` and even in that case when `use_cuda = False` the computation will still use a (CUDA-compatible) GPU if available, but will use inbuilt JAX primitives rather than the custom primitive being added in this PR. I think something like `use_healpix_custom_primitive` would be more a explicit name and changing docstring to something like

```
        use_healpix_custom_primitive (bool, optional): Whether to use a custom CUDA primitive for
            computing HEALPix fast fourier transform when `sampling = "healpix"` and running on a
            CUDA compatible GPU device. Using a custom primitive reduces long compilation times
            when JIT compiling. Defaults to `False`.
```